### ISSUE TITLE
Incorrect padding

### ISSUE THREAD

**Issue by YerongLi** [conceptofmind/toolformer#31]
When I when the training workflow with deepspeed in the readme, I met incorrect padding error. Does anyone knows how to fix this?
```
deepspeed train_gptj_toolformer.py --model_name_or_path=EleutherAI/gpt-j-6B --per_device_train_batch_size=4 \
  --num_train_epochs 10 --save_strategy=epoch --output_dir=finetune_toolformer_v0 --report_to "wandb" \
  --dataset_name dmayhem93/toolformer-v0-postprocessed --tokenizer_name customToolformer \
  --block_size 2048 --gradient_accumulation_steps 1 --do_train --do_eval --evaluation_strategy=epoch \
  --logging_strategy=epoch --fp16 --overwrite_output_dir --adam_beta1=0.9 --adam_beta2=0.999 \
  --weight_decay=2e-02 --learning_rate=1e-05 --warmup_steps=100 --per_device_eval_batch_size=1 \
  --cache_dir="hf_cache" --gradient_checkpointing=True --deepspeed ds_config_gpt_j.jso
```

```
    self.hf_deepspeed_config = HfTrainerDeepSpeedConfig(self.deepspeed)                                                                                                               [25/664]
  File "/scratch/yerong/.conda/envs/tool/lib/python3.9/site-packages/transformers/deepspeed.py", line 77, in __init__                                                                         
        super().__init__(config_file_or_dict)                                                                                                                                                 
  File "/scratch/yerong/.conda/envs/tool/lib/python3.9/site-packages/transformers/deepspeed.py", line 67, in __init__                                                                         
super().__init__(config_file_or_dict)                                                                                                                                                         
  File "/scratch/yerong/.conda/envs/tool/lib/python3.9/site-packages/transformers/deepspeed.py", line 67, in __init__                                                                         
    super().__init__(config_file_or_dict)                                                                                                                                                     
  File "/scratch/yerong/.conda/envs/tool/lib/python3.9/site-packages/accelerate/utils/deepspeed.py", line 52, in __init__                                                                     
    config_decoded = base64.urlsafe_b64decode(config_file_or_dict).decode("utf-8")                                                                                                            
  File "/scratch/yerong/.conda/envs/tool/lib/python3.9/base64.py", line 133, in urlsafe_b64decode                                                                                             
    super().__init__(config_file_or_dict)                                                                                                                                                     
  File "/scratch/yerong/.conda/envs/tool/lib/python3.9/site-packages/accelerate/utils/deepspeed.py", line 52, in __init__                                                                     
    config_decoded = base64.urlsafe_b64decode(config_file_or_dict).decode("utf-8")                                                                                                            
  File "/scratch/yerong/.conda/envs/tool/lib/python3.9/base64.py", line 133, in urlsafe_b64decode                                                                                             
    return b64decode(s)                                                                                                                                                                       
  File "/scratch/yerong/.conda/envs/tool/lib/python3.9/base64.py", line 87, in b64decode                                                                                                      
    return binascii.a2b_base64(s)   
binascii.Error:     Incorrect padding                                                                                                                                                         
return b64decode(s)
  File "/scratch/yerong/.conda/envs/tool/lib/python3.9/base64.py", line 87, in b64decode
    return binascii.a2b_base64(s)
binascii.Error: Incorrect padding
[2023-05-01 02:11:24,831] [INFO] [launch.py:428:sigkill_handler] Killing subprocess 19226
[2023-05-01 02:11:24,839] [INFO] [launch.py:428:sigkill_handler] Killing subprocess 19227
[2023-05-01 02:11:24,839] [ERROR] [launch.py:434:sigkill_handler] ['/scratch/yerong/.conda/envs/tool/bin/python', '-u', 'train_gptj_toolformer.py', '--local_rank=1', '--model_name_or_path=El
eutherAI/gpt-j-6B', '--per_device_train_batch_size=4', '--num_train_epochs', '10', '--save_strategy=epoch', '--output_dir=finetune_toolformer_v0', '--report_to', 'wandb', '--dataset_name', '
dmayhem93/toolformer-v0-postprocessed', '--tokenizer_name', 'customToolformer', '--block_size', '2048', '--gradient_accumulation_steps', '1', '--do_train', '--do_eval', '--evaluation_strateg
y=epoch', '--logging_strategy=epoch', '--fp16', '--overwrite_output_dir', '--adam_beta1=0.9', '--adam_beta2=0.999', '--weight_decay=2e-02', '--learning_rate=1e-05', '--warmup_steps=100', '--
per_device_eval_batch_size=1', '--cache_dir=hf_cache', '--gradient_checkpointing=True', '--deepspeed', 'ds_config_gpt_j.jso'] exits with return code = 1
```

**Comment by YerongLi** [comment]
I should have used "--deepspeed ds_config_gpt_j.json" instead of "--deepspeed ds_config_gpt_j.jso"

**Comment by belle9217** [comment]
> I should have used "--deepspeed ds_config_gpt_j.json" instead of "--deepspeed ds_config_gpt_j.jso"

hi，what do you mean? is  that different?
